.gitignore

```diff
@@ -28,6 +28,7 @@ out/
 *.orig
 geode-pulse/screenshots/
 /jpf.properties
+geode-assembly/assembly_content.txt
```

Comment:
Why is this change needed?

Reply:
This is where the AssemblyContentIntegrationTest dumps the file it generates (at least on my workstation). I copy this file to the src folder that it is supposed to reference, but this file itself I usually delete so that it doesn't get checked in. I thought I'd gitignore it so I don't have to keep deleting it.